lib/veteran_enrollment_system/associations/service.rb

```diff
@@ -66,7 +66,7 @@ def get_associations(form_id)
         end
       rescue => e
         StatsD.increment("#{STATSD_KEY_PREFIX}.get_associations.failed")
-        Rails.logger.error("#{form_id} retrieve associations failed: #{e.errors}")
+        Rails.logger.error("#{form_id} retrieve associations failed: #{e.detail}")
```

Comment:
There was an issue with some backend concern code breaking when using `errors`, so I changed it to `detail`.